README.md

```diff
@@ -30,6 +19,14 @@ These scripts simplify the process of:
 
 New user data is assumed to be in a Google Sheet. The function parse_rows in addusers.py handles parsing the spreadsheet data, you may need to modify it to work with your particular spreadsheet format.
 
+## Usage
+To run it:
+```
+# Possibly in a virtual environment 
+$ pip install -r requirements.txt 
```

Comment:
So, this does not need to be run every time you start the server, only during initial setup so that the dependencies are installed.  Can you make that clearer in the instructions?